README.md

```diff
@@ -20,7 +20,7 @@ Additionaly, you cat run TON proxy container for your gateway.
 ## Uploading content
 
 * Place necessary files to `exchange` folder.
-* Connect to docker container: `docker exec -it ton-storage-docker-gateway-1 bash`.
```

Comment:
The name of container depends on version and type of Compose (plugin or standalone).
Could you please change this line to:
`docker compose exec -it gateway bash`